src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -551,5 +551,215 @@ InModuleScope Arcus.Scripting.IntegrationAccount {
                 }
             }
         }
+        Context "Uploading Certificates into an Azure Integration Account" {
+            It "Try to upload single public certificate to unexisting Integration Account fails" {
```

Comment:
We only have a single error-test case. Can we come up with another? Seems like there's a lot of error handling.

Reply:
Just about all of the error handling is checked in the unit tests, for example the combinations of certain parameters that should or should not be be present. 

It's the same for the other objects (schemas, maps, assemblies) as well, they also have only one error-test case in the integration tests. I'll try to think of some more error-tests for the integration tests 🧠  but maybe we can pick that up in a separate PR, as the implementation for the certificates is the same as the already existing ones (schemas, maps, assemblies)?

Reply:
Yes, of course! 👍